src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -355,10 +355,6 @@ public string GetProjectProperty(Project dteProject, string propertyName, string
                 Debug.Assert(!ErrorHandler.Succeeded(hr) || hr != E_XML_ATTRIBUTE_NOT_FOUND,
                     $"Failed to get the property '{propertyName}' for project '{dteProject.Name}'.");
             }
-            else
-            {
-                Debug.Fail("Could not get IVsBuildPropertyStorage for EnvDTE.Project");
```

Comment:
I removed this debug failure. The alternative was to check if it's open as folder in the caller and avoid calling this method to begin with.